### ISSUE TITLE
FYI: enriched-text

### ISSUE THREAD

**Issue by alphapapa** [Kinneyzhang/rich-text#1]
Hi, this package looks cool, but you should probably be aware of the built-in Enriched Text mode:

+ `(find-file (expand-file-name "enriched.txt" data-directory))`
+ `(info "(emacs) Enriched Text")`

Maybe you could contribute to some the longstanding items on its to-do list.  :)

**Comment by Kinneyzhang** [comment]
Definitely, I will research on this cool package. Thanks for your sharing!
